src/components/SummaryDetail/SummaryDetail.js

```diff
@@ -10,32 +10,36 @@ const SummaryDetail = ({
   onOpen,
   title,
   renderTitle,
-}) => (
-  <div className={cx('slds-summary-detail', { 'slds-is-open': isOpen })}>
-    {onOpen && (
-      <IconButton
-        onClick={onOpen}
-        className={cx('slds-m-right_x-small', iconButtonClassName)}
-      >
-        <ButtonIcon
-          className="slds-summary-detail__action-icon"
-          icon="switch"
-          sprite="utility"
-        />
-      </IconButton>
-    )}
-    <div>
-      <div className="slds-summary-detail__title">
-        {renderTitle(title)}
-      </div>
-      {children && (
-        <div aria-hidden={!isOpen} className="slds-summary-detail__content">
-          {children({ isOpen })}
-        </div>
+}) => {
+  if (!onOpen) return renderTitle(title);
```

Comment:
This seems to directly contradict the example on https://lightningdesignsystem.com/components/summary-detail/#Closed

Reply:
In the specifications the `Base` and the `With Step Progress` layout of the `SetupAssistant` do not have the `slds-summary-detail` wrapper. It's only needed for the `Hub with Expandable Steps`.

Reply:
The initial problem that lead me to this fix was that the actions on the right were left aligned if there was less text. Following the specifications the actions should be right aligned.
<img width="276" alt="Screenshot 2020-01-17 at 11 27 53" src="https://user-images.githubusercontent.com/511551/72605169-8ca96280-391c-11ea-8dd2-563b56027ab2.png">

Reply:
That all makes sense in the case of `SummaryDetail` being used in a `SetupAssistant`. However, the documentation for SummaryDetail outlines different markup for both the base component and the closed state. Since `SummaryDetail` is a standalone component, that standalone component now uses different markup than documented. In my eyes, this fix only applies to `SetupAssistant` while potentially breaking the standalone component.

Reply:
Makes sense, I will solve this issue within the `SetupAssistant` component.